.github/workflows/no-response.yml

```diff
@@ -17,16 +17,25 @@ jobs:
       - name: 🥺 Handle Ghosting
         uses: actions/stale@v8
         with:
-          days-before-close: 21
           close-issue-message: >
-            This issue has been automatically closed because we haven't received a
-            response from the original author 🙈. This automation helps keep the issue
-            tracker clean from issues that are unactionable. Please reach out if you
-            have more information for us! 🙂
+            This issue has been automatically closed because we didn't hear
+            anything from the original author after the previous notice anymore.
           close-pr-message: >
-            This PR has been automatically closed because we haven't received a
-            response from the original author 🙈. This automation helps keep the issue
-            tracker clean from PRs that are unactionable. Please reach out if you
-            want to resume the work on this PR! 🙂
+            This PR has been automatically closed because we didn't hear
+            anything from the original author after the previous notice anymore.
```

Comment:
```suggestion
            This PR has been automatically closed because we didn't hear
            anything from the original author after the previous notice.
```